lib/memory/index.js

```diff
@@ -58,13 +58,13 @@ module.exports = function MemoryStore(data_set) {
 
   const update = R.curry((bucket, id, data) =>
 
-      getById(bucket, id)
+    getById(bucket, id)
 
     .tap(R.when(R.isNil, () => throwItemNotFound(bucket, id)))
 
     .then(() => { store[bucket][id] = data })
 
-    .return(id)
+    .return(1)
```

Comment:
the mysql update returns the number of affected rows. since it updates by `id` it's only possible to update 1 row.